authui/src/authflowv2/components/checkbox.css

```diff
@@ -0,0 +1,64 @@
+@layer components {
+  :root {
+    --checkbox__size: 1.125em;
+    --checkbox__border-radius: 0.25em;
+    --checkbox__ring-width: 1.5px;
+    --checkbox__ring-color: var(--color-grey-200);
+    --checkbox__ring-color-checked: var(--color-primary-base);
+    --checkbox__background-color: var(--color-white);
+    --checkbox__background-color--checked: var(--color-primary-base);
+
+    --checkbox__check-color: var(--color-white);
+  }
+
+  :root.dark {
+    /* TODO(newman): Dark mode */
+  }
+
+  .checkbox {
+    @apply relative;
+    @apply inline-block;
+
+    width: var(--checkbox__size);
+    height: var(--checkbox__size);
+
+    input {
+      @apply absolute;
+      @apply opacity-0;
+      @apply w-0;
+      @apply h-0;
+
+      &:checked + label {
+        @apply bg-[color:var(--checkbox\_\_background-color--checked)];
+        @apply ring-[color:var(--checkbox\_\_ring-color-checked)];
+      }
+
+      &:checked + label::before {
+        @apply absolute;
+        @apply inset-0;
+        @apply text-center;
+        @apply leading-[var(--checkbox\_\_size)];
+
+        content: "check";
+        font-family: "material-icons";
```

Comment:
The .material-icons class does more than setting font-family. Use `<span class="material-icons">check</span>` instead.